docs/secure-connections/sauce-connect-5/faq.md

```diff
@@ -6,147 +6,57 @@ sidebar_label: FAQ
-
-## (CIA) If we have five users, should we use five instances of Sauce Connect Proxy or set up one shared instance?
-
-Feel free to do either, even if you only have one Sauce account.
-
-If you do decide to use five separate instances, you'll need to create unique identifiers for each. More information: **High Availability Sauce Connect 5 Setup needed**.
+To improve Sauce Connect 5 tunnel performance, start by ensuring that the instance running Sauce Connect has sufficient system resources. You can find recommended CPU and memory requirements in [System Requirements](/secure-connections/sauce-connect-5/system-requirements/#cpu-and-memory-resources).
 
-## (CIA) Can I split testing traffic across multiple Sauce Connect Proxy tunnels?
+Another effective way to optimize performance is to reduce unnecessary traffic passing through the tunnel. You can use the [-F, --deny-domains](/dev/cli/sauce-connect-5/run#deny-domains) flag to block domains that do not need to be accessed during testing. This helps limit bandwidth usage and reduces load on the tunnel.
 
-Sauce Connect Proxy offers some options for high availability that enable multiple Sauce Connect Proxy tunnels to be grouped into a pool, which is treated like a single tunnel. This has several advantages, including the ability to distribute the load of test traffic. More information: **High Availability Sauce Connect 5 Setup needed**.
+## How many tests can I run in each Sauce Connect instance?
 
-## (CIA) Does Sauce Connect Proxy support running XCUITest and Espresso tests?
+The number of tests isn't the limiting factor — what matters is the amount of traffic those tests generate. Sauce Connect 5 performance depends on system resources and the number of concurrent HTTP requests.
 
-Yes. More information: [Mobile Testing Frameworks](/mobile-apps/automated-testing/appium).
+As noted in the [system requirements](/secure-connections/sauce-connect-5/system-requirements), a single instance with 4 CPU cores can typically handle up to 4096 concurrent HTTP requests. To determine how many tests your instance can support, estimate the expected number of concurrent requests during peak test execution and scale accordingly.
 
-## How many Sauce Connect Proxy tunnels can I keep open simultaneously?
+## How many Sauce Connect instances can I keep open simultaneously?
 
-Your options depend on your license type. More information: [System and Network Requirements for Sauce Connect Proxy](/secure-connections/sauce-connect-5/system-requirements).
+The number of concurrent Sauce Connect instances you can run depends on your Sauce Labs contract. By default, the limit is 5 active instances, but this can be increased based on your plan. Contact your Sauce Labs representative if you need a higher limit.
```

Comment:
@budziam Do you happen to know about the limits?

Reply:
min:5, max: 30, although max doesn't apply to all the customers

Reply:
Thanks! I've included the limits in the answer.